tests/server/test_compile_details.py

```diff
@@ -86,6 +85,10 @@ async def env_with_compiles(client, environment):
     return environment, ids, compile_requested_timestamps
 
 
+def parse_datetime_to_utc(time: str) -> datetime.datetime:
+    return datetime.datetime.strptime(time, "%Y-%m-%dT%H:%M:%S.%f").replace(tzinfo=datetime.timezone.utc)
```

Comment:
This is a handy method that will be useful in other places in the future. Let's anticipate on that and put it into the util package.